servers/biometric_signal_sensor_interface/src/anomaly_detector/anomaly_detector.py

```diff
@@ -151,49 +151,52 @@ def vt_anomaly_detect(self, ecg, rr_intervals,
 
             Refer to readme for more details
         """
-        __zero_one_count = True
-
-        VTobj = VentricularTachycardia(ecg, rr_intervals,
-                                       rr_interval_status, self.config)
-
-        further_analyze = VTobj.analyze_six_second()
-        # if initial analysis indicates that further analysis
-        # is not required
-        if not further_analyze:
-            __zero_one_count = False
-            self.vt_result = __zero_one_count
-
-        # the print can be commented out
-        print("Doing further analysis")
-
-        # perform the preprocessing
-        VTobj.signal_preprocess()
-
-        # call the DangerousHeartActivity detector
-        cur_ampl, stop_cur = VTobj.DHA_detect(prev_ampl)
-
-        # whatever be the results of the following stages,
-        # we necessarily have to analyze the next six second epoch
-
-        # if further analysis is not required
-        if stop_cur is True:
-            self.vt_result = __zero_one_count
-
-        # asystole detector
-        vtvfres = VTobj.asystole_detector(cur_ampl)
-
-        # to analyze next six second epoch
-        if vtvfres == 'VT/VF':
-            # A VT episode has been found
-            # the print can be omitted
-            print(vtvfres)
-            __zero_one_count = VTobj.zero_one_count
-            self.vt_result = __zero_one_count
-        else:
-            # not a VT episode
-            # the print can be omitted
-            print(vtvfres)
-            self.vt_result = __zero_one_count
+        try:
```

Comment:
I'm leaving this to Dipankar as he is the author.
I wouldn't want to mess his side up or break it.